PlainTasks.py

```diff
@@ -524,9 +538,12 @@ def show_panel_or_open(self, fn, sym, line, col, text):
                 for root, _, _ in os.walk(folder):
                     name = os.path.join(root, fn)
                     if os.path.isfile(name):
-                        self._current_res.append((name, line or 0, col or 0))
-            if os.path.isfile(fn): # check for full path
-                self._current_res.append((fn, line or 0, col or 0))
+                        self._current_res.append((name, line or 0, col or 0, "f"))
+            if os.path.isfile(fn):  # check for full path
+                self._current_res.append((fn, line or 0, col or 0, "f"))
+            elif os.path.isdir(fn):
+                # self._open_folder(fn)
+                self._current_res.append((fn, 0, 0, "d"))
```

Comment:
But what about relative paths?
AFAIU, `fn` here is path as it written in users todo-file so it will work if user wrote full path.
To support relative one you should modify for-loop started at line 538.